test/test.sh.in

```diff
@@ -63,7 +63,8 @@ for t in $@; do
       fail_count=$((fail_count+1))
       ;;
     *)
-      echo "Test harness error" 1>&2
+      echo "Test harness error: ${t} w/ MALLOC_CONF=\"${MALLOC_CONF}\"" 1>&2
+      echo "Use prefix to debug, e.g. JEMALLOC_TEST_PREFIX=\"gdb --args\" sh test/test.sh ${t}" 1>&2
```

Comment:
Oh we are invoking the test through "sh test/test.sh", so the path should be included from the script?